miio/integrations/vacuum/dreame/dreamevacuum_miot.py

```diff
@@ -46,14 +115,22 @@ class ChargingState(Enum):
     GoCharging = 5
 
 
-class CleaningMode(Enum):
+class CleaningModeDreame1C(Enum):
     Unknown = -1
     Quiet = 0
     Default = 1
     Medium = 2
     Strong = 3
 
 
+class CleaningModeDreameF9(Enum):
+    Unknown = -1
+    Quiet = 0
+    Standart = 1
+    Strong = 2
+    Turbo = 3
```

Comment:
This is similar to fan speed setting on other vacuum integrations, right? Following the API from roborock integration (`fan_speed_presets() -> Dict[str, int]` to return the available modes) is the way to go. You can check the model inside the class using `self.model` to find out which presets to return.